CMakeLists.txt

```diff
@@ -141,22 +142,27 @@ find_package(Qt6QTiffPlugin 6.5.3 REQUIRED PATHS ${Qt6Gui_DIR})
 
 # copy all dlls into place.
 if(WIN32)
-  # find_package(Qt6QWindowsIntegrationPlugin 6.5.3 REQUIRED PATHS ${Qt6Gui_DIR})
-  # find_package(Qt6QWindowsVistaStylePlugin 6.5.3 REQUIRED PATHS ${Qt6Gui_DIR})
-
-  # assuming vcpkg has done some work already to get dlls in here
-  install(
-    DIRECTORY ${CMAKE_BINARY_DIR}/Release/
-    DESTINATION ${INSTALLDIR}
-    CONFIGURATIONS Release
-    FILES_MATCHING PATTERN "*.dll"
-  )
-  install(
-    DIRECTORY ${CMAKE_BINARY_DIR}/Debug/
-    DESTINATION ${INSTALLDIR}
-    CONFIGURATIONS Debug
-    FILES_MATCHING PATTERN "*.dll"
+  install(CODE [[
+  file(GET_RUNTIME_DEPENDENCIES
+    EXECUTABLES "$<TARGET_FILE:agaveapp>"
+    RESOLVED_DEPENDENCIES_VAR _r_deps
+    UNRESOLVED_DEPENDENCIES_VAR _u_deps
+    DIRECTORIES "$<TARGET_FILE_DIR:agaveapp>"
+    PRE_EXCLUDE_REGEXES "api-ms-" "ext-ms-"
+    POST_EXCLUDE_REGEXES ".*system32/.*\\.dll"
   )
+  foreach(_file ${_r_deps})

```

Comment:
the new code above this foreach collects all the runtime dependencies (aka DLLs on Windows) of the target agaveapp (which is agave's executable itself).  It excludes dependencies that are ms windows builtin or standard runtimes.

Then the foreach actually makes sure those dependencies are copied into the right place for the installable artifact.